client/src/test/java/com/spectralogic/tpfr/client/ReadFileHelper.java

```diff
@@ -0,0 +1,27 @@
+package com.spectralogic.tpfr.client;
+
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+final class ReadFileHelper {
+    private static final Logger LOG = LoggerFactory.getLogger(ClientImpl.class);
+
+    static String readFile(final String fileName){
+
+        String result = "";
```

Comment:
`result` does not need to be mutable. You can just return from within the try block, and return an empty string from the catch block.